.github/workflows/custom_helpline.yml

```diff
@@ -0,0 +1,120 @@
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: '${{ secrets.AWS_DEFAULT_REGION }}'
+      - name: Set IP_FIND_API_KEY
+        uses: "marvinpinto/action-inject-ssm-secrets@latest"
+        with:
+          ssm_parameter: "IP_FIND_API_KEY"
+          env_variable_name: "IP_FIND_API_KEY"
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: '${{ secrets.AWS_DEFAULT_REGION }}'
+      - name: Set SERVERLESS_URL
+        uses: "marvinpinto/action-inject-ssm-secrets@latest"
+        with:
+          ssm_parameter: "/${{ inputs.environment }}/serverless/${{ inputs.account-sid }}/base-url"
+          env_variable_name: "SERVERLESS_URL"
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: '${{ secrets.AWS_DEFAULT_REGION }}'
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+      # Create and fill secret.ts
+      - name: Create secret.ts
+        run: |
+          touch ./private/secret.ts
+      - name: Fill secret.ts
+        run: |
+          cat <<EOT >> ./private/secret.ts
+          export const API_KEY = '${{ env.IP_FIND_API_KEY }}';
+          export const SERVERLESS_URL = '${{ env.SERVERLESS_URL }}';
+          EOT
+      # Run a single command using the runners shell to install dependencies 
+      - name: Install dependencies
+        run: npm install
+      # Build and compile Webchat
+      - name: Run build command
+        run: npm run build:${{ inputs.build-env }}
+      # Rename the builded js file
+      - name: Rename js
+        run: mv bundle.js aselo-chat.min.js
+        working-directory: ./build
+      # Upload Webchat to S3 bucket
+      - name: Upload Webchat to S3 bucket
+        uses: jakejarvis/s3-sync-action@master
+        with:
+          args: --acl public-read --follow-symlinks --exclude '*' --include '*.js' --content-type 'application/javascript; charset=utf-8'
+        env:
+          AWS_S3_BUCKET: ${{ inputs.aws-s3-bucket }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: '${{ secrets.AWS_DEFAULT_REGION }}'
+          SOURCE_DIR: 'build'
+
+      # Send Slack notifying success
+      - name: Slack Aselo channel
+        id: slack
+        uses: slackapi/slack-github-action@v1.14.0
+        with:
+          channel-id: ${{ env.ASELO_DEPLOYS_CHANNEL_ID }}
+          slack-message: "`[WEBCHAT]` Action ${{ github.workflow }} completed with SHA ${{ github.sha }} :rocket:."
```

Comment:
Can we include additional info in here: 
environment & helpline code being deployed (input.build-env if you don't do the suggested refactor)
triggering actor (will be git user name for manually triggered deploys)
branch / tag being deployed

You should copy the formatting & content used in flex: https://github.com/techmatters/flex-plugins/blob/8721d2ec21adef7c2e4077df05468e0dce997df7/.github/actions/main-action/action.yml#L134